html/src/components/terminal/xterm/index.ts

```diff
@@ -378,6 +378,15 @@ export class Xterm {
                 case 'isWindows':
                     if (value) console.log('[ttyd] is windows');
                     break;
+                case 'fontFamily': {
+                    const fontSize = prefs['fontSize'] || '13';
+                    document.fonts.load(`${fontSize}px "${value}"`).then(() => {
```

Comment:
is the load operation really needed? you can already set font family for xterm without this change (via code in the default block).

Reply:
Back when I originally wrote this it was needed to ensure the font was present on the first page load on a system without the font. I removed the font from my system and ran through some tests and it seems this is no longer needed. With the font available the code in the default block is adequate.